_posts/2016-07-07-Rust-1.10.md

```diff
@@ -0,0 +1,270 @@
+panic=abort` flag] or a [setting in your `Cargo.toml`], when a `panic!` happens,
+it will abort, rather than unwind. Why would you want to do this? With an
+abort, less code gets generated, meaning that binary sizes are a bit smaller,
+and compilation time is ever-so-slightly faster. You give up the ability to
+recover, however.
+
+[`-C panic=abort` flag]: https://github.com/rust-lang/rust/pull/32900
+[setting in your `Cargo.toml`]: https://github.com/rust-lang/cargo/pull/2687
+[RFC 1513]: https://github.com/rust-lang/rfcs/blob/master/text/1513-less-unwinding.md
+
+The second big feature in 1.10 is a new crate type: [`cydlib`]. Rust already has
+one dynamic library format, `dylib`. These libraries are currently used everywhere
+you need a dynamic library written in Rust, but as it turns out, there are two major
+use-cases for them: writing a dynamic library to be used in a Rust project, and writing
+a dynamic library to be embedded in some other language. The `dylib` format is focused
+on the first case, and contains extraneous information that's unimportant for the second,
+such as Rust-specific metadata. One final note about this feature: support is in the Rust
+compiler, but not yet in Cargo.
+
+[`cdylib`]: https://github.com/rust-lang/rust/pull/33553
+[RFC 1510]: https://github.com/rust-lang/rfcs/blob/master/text/1510-rdylib.md
+
+[RFC 1521] has landed. This RFC ensures that every type that is `Copy` also [has
+a `Clone` implementation]. Given that `Copy` is an implicit copy, `Clone` is an explicit
+copy, the fact that some types which were `Copy` but not `Clone` felt wrong: if you can
+implicitly copy something, you should be able to also call it explicitly. (But not the
+reverse: many types are `Clone` but not `Copy`.) This has now been rectified.
+
+[has a `Clone` implementation]: https://github.com/rust-lang/rust/pull/33420
+[RFC 1521]: https://github.com/rust-lang/rfcs/blob/master/text/1521-copy-clone-semantics.md
+
+In addition, [a number of performance improvements landed in the
+compiler](https://github.com/rust-lang/rust/blob/master/RELEASES.md#performance),
+and so did [a number of usability
+improvements](https://github.com/rust-lang/rust/blob/master/RELEASES.md#usability)
+across the documentation, `rustdoc` itself, and various error messages.
+
+Finally, there's a large change to the way that we develop Rust that won't
+impact Rust users directly, but will help those distributing Rust
+significantly. Rust is implemented in Rust, which means that to build a copy of
+Rust, you need a copy of Rust. This is commonly referred to as 'bootstrapping'.
+Historically, we would do this by choosing a specific version of the compiler,
+and then updating it periodically.  Furthermore, since the Rust compiler uses
+unstable Rust features, in order to build a copy of the stable compiler, you
+would need a specific version of the Rust compiler. This has served us well for
```

Comment:
Should “specific version” be “nightly version” here? Otherwise this seems to be repeating the previous sentence.